docs/CONTRIBUTING.md

```diff
@@ -124,6 +130,32 @@ or alternatively:
 npx jest --watch
 ```
 
+## Deploy and test to a hosted Twilio serverless environment
+1. Create a profile/api key, if you don't already have one
+```
+twilio login
+```
+2. List existing profiles
+```
+twilio profiles:list
+```
+2. Activate the profile
+```
+twilio profiles:use <your_profile_id>
+```
+3. Deploy
+```twilio serverless:deploy```
+
+## Creating a pull request
```

Comment:
This moves the pull request section between two sections on testing a template (above "Deploy and test to a hosted Twilio serverless environment" and below "Testing your template").

This whole section could maybe use a hierarchy update with a level 2 heading of "Testing" and level 3 sub-heads for "Testing the functionality of your new template locally", "Running tests" (could probably be "Running automated tests"), "Deploy and test to a hosted Twilio serverless environment" and "Testing your template" (could also be updated to "Test the installation of your template"). Then "Creating a pull request" could be a  new level 2 heading.

E.g.

## Testing

### Testing the functionality of your new template locally

1. Make sure you have the [Twilio CLI](https://www.twilio.com/docs/twilio-cli/quickstart) installed.

etc.

### Running automated tests

The tests are written using [Jest](https://jestjs.io/). You can run the test suite by running:

etc.

### Deploy and test to a hosted Twilio serverless environment

 1. Create a profile/api key, if you don't already have one

etc.

### Test the installation of your template

If you want to test how your new template works with the Twilio CLI...

etc.

## Creating a pull request

Please open a pull request on the [function-templates](https://github.com/twilio-labs/function-templates/pulls) repository...

etc.